.env.test

```diff
@@ -1,11 +0,0 @@
-# Flask server configuration.
```

Comment:
Why have we deleted this? We should keep it around for running integration tests (at least locally) unless we're planning to update the instructions for those to explain how to set the environment variables manually